core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.controller.ts

```diff
@@ -193,14 +196,16 @@ angular.module('oppia').controller(
       };
 
       $scope.rejectAndReviewNext = function(reviewMessage) {
-        $scope.resolvingSuggestion = true;
-        SiteAnalyticsService.registerContributorDashboardRejectSuggestion(
-          'Translation');
-
-        ContributionAndReviewService.resolveSuggestionToExploration(
-          $scope.activeSuggestion.target_id, $scope.activeSuggestionId,
-          ACTION_REJECT_SUGGESTION, reviewMessage || $scope.reviewMessage,
-          generateCommitMessage(), $scope.showNextItemToReview);
+        if (ValidatorsService.isValidReviewMessage(reviewMessage, true)) {
```

Comment:
Nit: Add an inline comment indicating the boolean parameter so that it's obvious what `true` means from the call site, i.e. 

```suggestion
        if (ValidatorsService.isValidReviewMessage(reviewMessage, /* showWarnings= */ true)) {
```

Reply:
Done.